### PR TITLE
[Form] Remove the scale argument of the IntegerToLocalizedStringTransformer

### PR DESCRIPTION
**Pull request by yceruto**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

See https://github.com/symfony/symfony/pull/28569

### CODE REVIEW

File: src/Symfony/Component/Form/Extension/Core/DataTransformer/IntegerToLocalizedStringTransformer.php [hunk 1]
```diff
@@ -27,15 +27,8 @@ class IntegerToLocalizedStringTransformer extends NumberToLocalizedStringTransfo
      * @param bool $grouping     Whether thousands should be grouped
      * @param int  $roundingMode One of the ROUND_ constants in this class
      */
-    public function __construct($grouping = false, $roundingMode = self::ROUND_DOWN)
+    public function __construct(?bool $grouping = false, int $roundingMode = self::ROUND_DOWN)
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Form/Extension/Core/DataTransformer/IntegerToLocalizedStringTransformer.php]: $roundingMode can be null in the parent

**Comment by nicolas-grekas** [comment]
(another rebase needed :) )

**Comment by yceruto** [comment]
this is the last one for now :)

**Comment by nicolas-grekas** [comment]
Thank you @yceruto.

**Pull request merged**